src/devices/Rfid/Mifare/Mifare.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
I feel like we should not subsplit Rfid project. Just one top project seem good enough I think (especially it ends up being in Iot.Device.Bindings anyway)

Reply:
I made few modifications as I've been playing as well with a USB smart card reader:
- I renamed the main Rfid folder to Card
- I moved the Rfid elements under a folder called Rfid which is specific to Rfid
- Mifare card can be with a chip as well
- I've been working on a Credit Card project so I'll add the Credit Card for the next commit. This does allow to read all available data from the Credit Card. This part is working as well with a normal USB reader. Just need to pass a reader (rfid or smart card) that implement the CardWriteRead class with the function WriteRead and all is working the same way.
- I regrouped Rfid specific and Card transversal 
Reason why:
- Readers are independ from the cards (Mifare, Credit Card and others we will add later on)
- Readers have a depency on Rfid or Card or both